src/main/resources/alfresco/module/alfresco-gcs-connector/alfresco-global.properties

```diff
@@ -16,34 +16,4 @@ gcs.bucketName.deleted=${gcs.bucketName}
 
 #path in bucket e.g.: contenstore will store objects in gcs://contenststore/2019/...
 gcs.dir.contentstore=${dir.contentstore}
-gcs.dir.contentstore.deleted=${dir.contentstore.deleted}
-
-dir.cachedcontent=/path/to/cache
-dir.root=/path/to/local
-
-##############################
-# Caching default properties #
-##############################
-
-system.content.caching.cacheOnInbound=true
-system.content.caching.maxUsageMB=4096
-# maxFileSizeMB - 0 means no max file size.
-system.content.caching.maxFileSizeMB=0
-# When the CachingContentStore is about to write a cache file but the disk usage is in excess of panicThresholdPct
-# (default 90%) then the cache file is not written and the cleaner is started (if not already running) in a new thread.
-system.content.caching.panicThresholdPct=90
-# When a cache file has been written that results in cleanThresholdPct (default 80%) of maxUsageBytes
-# being exceeded then the cached content cleaner is invoked (if not already running) in a new thread.
-system.content.caching.cleanThresholdPct=80
-# An aggressive cleaner is run till the targetUsagePct (default 70%) of maxUsageBytes is achieved
-system.content.caching.targetUsagePct=70
-# Threshold in seconds indicating a minimal gap between normal cleanup starts
-system.content.caching.normalCleanThresholdSec=0
-system.content.caching.minFileAgeInMillis=2000
-system.content.caching.maxDeleteWatchCount=1
-# Clean up every day at 3 am
-system.content.caching.contentCleanup.cronExpression=0 0 3 * * ?
-system.content.caching.timeToLiveSeconds=0
-system.content.caching.timeToIdleSeconds=60
-system.content.caching.maxElementsInMemory=5000
-system.content.caching.maxElementsOnDisk=10000
```

Comment:
Was the caching also being picked up from here like the dir.root?

Reply:
I see the alfresco-global.properties file in a module as a place to set good defaults for that module. In this case especially for the gcs.* properties.

In that regard the "/path/to/..." doesn't make a lot of sense as a default.
Alfresco's defaults for dir.root from repository.properties and system.content.caching.* should be ok in my opinion.

I don't know why it picked up those values over the alfresco-global.properties one. It should according to core-services-context.xml:
<property name="locations">
            <list>
                <value>classpath:alfresco/repository.properties</value>
                <value>classpath:alfresco/domain/transaction.properties</value>
                <value>classpath:alfresco/caches.properties</value>
                <!-- Defaults for "decomposed" non-enterprise projects, e.g. alfresco-remote-api -->
                <value>classpath*:alfresco/project-*.properties</value>
                <!-- Enterprise defaults -->
                <!--  Overrides supplied if this is an enterprise install (none exist for community) -->
                <value>classpath*:alfresco/enterprise/caches.properties</value>
                <value>classpath*:alfresco/enterprise/repository.properties</value>
                <!-- <value>classpath:alfresco/jndi.properties</value> -->
                <!--  Overrides supplied by modules -->
                <value>classpath*:alfresco/module/*/alfresco-global.properties</value>
                <!--  Installer or user-provided defaults -->
                <value>classpath*:alfresco-global.properties</value>
            </list>
        </property>